helpdesk/tests/test_webhooks.py

```diff
@@ -116,6 +124,35 @@ def test_create_ticket_and_followup_via_api(self):
         self.assertEqual(handled_webhook_requests['new_ticket_requests'][-1]["ticket"]["title"], "Test title")
         self.assertEqual(handled_webhook_requests['new_ticket_requests_1'][-1]["ticket"]["title"], "Test title")
         self.assertEqual(handled_webhook_requests['new_ticket_requests'][-1]["ticket"]["description"], "Test description\nMulti lines")
+        ticket = Ticket.objects.get(id=handled_webhook_requests["new_ticket_requests"][-1]["ticket"]["id"])
+        ticket.set_custom_field_values()
+        serializer = TicketSerializer(ticket)
+        self.assertEqual(
+            list(sorted(serializer.fields.keys())),
+            ['assigned_to',
+             'attachment',
+             'custom_my_custom_field',
+             'description',
+             'due_date',
+             'followup_set',
+             'id',
+             'merged_to',
+             'on_hold',
+             'priority',
+             'queue',
+             'resolution',
+             'status',
+             'submitter_email',
+             'title']
+        )
+        with open("/tmp/request", "w") as fd:
+            import json
+            json.dump(handled_webhook_requests['new_ticket_requests_1'][-1]["ticket"], fd, indent=4, default=str)
+        with open("/tmp/serilaizer_data", "w") as fd:
+            import json
+            json.dump(serializer.data, fd, indent=4, default=str)
```

Comment:
Were these lines useful for debugging? I don't see what they bring to the test

Reply:
Oops. sorry about that, will remove. Yes I was trying to figure out what was going on...

Reply:
Will merge when the extra code is removed